io-template.yml

```diff
@@ -2,10 +2,10 @@
 
 variables:  
   ASSET_ID: $CI_PROJECT_PATH
-  WORKFLOW_ENGINE_VERSION: "2021.04.2"
+  WORKFLOW_ENGINE_VERSION: "2021.06"
```

Comment:
The workflow engine version should be 2021.6.0

Reply:
Updated with merged PR - https://github.com/synopsys-sig/io-artifacts/pull/25